tin-bachproef/abstract.tex

```diff
@@ -0,0 +1,20 @@
+\begin{abstract}
+% TODO: De "abstract" of samenvatting is een kernachtige (max 1 blz. voor een
+% thesis) synthese van het document. In ons geval beschrijf je kort de
+% probleemstelling en de context, de onderzoeksvragen, de aanpak en de
+% resultaten.
+% \lipsum[1-4]
+De informatica-business staat niet stil. Dat weet men van het moment dat men er de eerste stappen in zet. Een welbekende quote uit de IT-wereld, \emph{The one certainty is change.}, is hier het levende bewijs van. Een ontwikkelaar moet daarom vrede nemen met het feit dat hij in deze branche levenslang zal moeten bijleren.
+\\
+De IT-business evolueert constant. Gebruikers passen zich aan aan nieuwe systemen die op de markt komen en verwachten van de ontwikkelaar dat hij zich hier ook aan aanpast, en de tijdlijn van applicatieontwikkeling is hier het beste voorbeeld van.
+\\
+Vroeger was het aanleveren van oplossingen aan de klant een gemakkelijkere zaak. Applicaties werden geschreven in één enkele codetaal en konden op één soort client draaien, in bijna elk geval een desktop-machine. De gehele applicatie werd gebundeld in één consistent pakket: Een monolithische applicatie die alle logica en functionaliteit bevatte. De dag van vandaag hebben we echter te maken met een ontwikkeling in de business waar dit soort architectuur niet tegen opgewassen is. 
```

Comment:
Vandaag de dag hebben we echter te maken met recente ontwikkelingen waar dit soort architectuur niet tegen opgewassen is.